scripts/export/v0_33_0/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import json
+import math
+import sys
+
+
+# default_date = datetime.datetime(year=2019, month=2, day=11, hour=12, tzinfo=datetime.timezone.utc) # '2019-02-11T12:00:00Z'
+
+parser = argparse.ArgumentParser(description='Convert genesis.json from v0.33.2 to v0.34.0')
+parser.add_argument('exported_genesis', type=argparse.FileType('r'), default=sys.stdin)
+parser.add_argument('--chain-id', type=str, default='cosmoshub-n')
+parser.add_argument('--start-time', type=str, default='2019-02-11T12:00:00Z')
+args = parser.parse_args()
+
+chain_id = args.chain_id.strip()
+
+if chain_id == '':
+  print('chain-id required')
+  sys.exit(1)
```

Comment:
You can make these 2 lines more compact:

```suggestion
  sys.exit('chain-id required')
```